src/components/ExperimentDetail.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState } from 'react'
+// import { Link } from 'react-router-dom'
+import { Container, Row, Col, Alert, Card, Progress, CardBody, CardText, FormGroup, Label, Input, CustomInput, DropdownToggle, DropdownMenu, DropdownItem, UncontrolledButtonDropdown, Nav, NavItem, NavLink, TabContent, TabPane } from "reactstrap";
+import "../static/css/Experiments.css";
+import classnames from 'classnames';
+ 
+
+const ExperimentDetail = () => {
+
+    const data = [
+        { id: "1", emojis: "🚀💸", name:"Up Only WBTC" , contract: "OxSalo12345", assetPrice: "1.00", currency: "$", totalAssets: "1000", totalAUM: "10000", baseAsset: "WBTC"},
+      ];
+
+    const [activeTab, setActiveTab] = useState('1');
+
+    const toggle = (tab: any) => {
+        if(activeTab !== tab) setActiveTab(tab);
+    }
+
+    const navItems = ['Details', 'Wallet', 'Contracts']
```

Comment:
How do you feel about splitting up each tab content into its own component? That way it's smaller and easier to manage.

Reply:
great suggestion, will proceed to do so!